app/js/components/orderEntry/OrderHeader.jsx

```diff
@@ -1,13 +1,22 @@
 import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
+import IconButton from '../button/IconButton';
 
 const OrderHeader = ({ status, orderable }) => (
   <Fragment>
     <th>{orderable}</th>
     <th>{status}</th>
-    <th>
-      <i className="icon-edit" />
-      <i className="icon-delete" />
+    <th className="draft-action-btns">
+      <IconButton
+        iconClass="icon-pencil"
+        iconTitle="EDIT"
+        onClick={() => {}}
+      />
+      <IconButton
+        iconClass="icon-remove"
+        iconTitle="DISCARD"
+        onClick={() => {}}
+      />
```

Comment:
From the component composition above, I see you rendered edit/discontinue icon for both lab-orders and drug-orders. I don't think this is supposed to be, the icon rendering has to be dynamic, edit/discontinue-icon for drug orders while urgency-icon for lab orders.

Reply:
@IAMOTZ the implementation on this ticket followed the designs from the mockup and it wasn't stated on the ticket either, but after further clarification from David, he said to make the lab-orders non-editable for now, so not even urgency toggling for now which I have fixed on OEUI-225 which is a sequel to this.

Reply:
Okay.
Thanks for the clarification. 👍